src/components/LanguageListItemCard/LanguageListItemCard.tsx

```diff
@@ -0,0 +1,81 @@
+import { Card, CardContent, makeStyles, Typography } from '@material-ui/core';
+import clsx from 'clsx';
+import { FC } from 'react';
+import * as React from 'react';
+import { LanguageListItemFragment } from '../../api';
+import { KeyValProp } from '../KeyValProp';
+import { CardActionAreaLink } from '../Routing';
+
+const useStyles = makeStyles(({ spacing }) => {
+  return {
+    root: {
+      width: '100%',
+      maxWidth: '400px',
+    },
+    cardContent: {
+      display: 'flex',
+      justifyContent: 'space-between',
+    },
+    rightContent: {
+      display: 'flex',
+      flexDirection: 'column',
+      justifyContent: 'flex-end',
+    },
+    location: {
+      marginBottom: spacing(2),
+    },
+  };
+});
+
+export type LanguageListItemCardProps = LanguageListItemFragment & {
+  className?: string;
+};
+
+export const LanguageListItemCard: FC<LanguageListItemCardProps> = ({
+  className,
+  name,
+  ethnologueName,
+  rodNumber,
+  ethnologuePopulation,
+}) => {
+  const classes = useStyles();
+
+  return (
+    <Card className={clsx(classes.root, className)}>
+      <CardActionAreaLink to="/some-path">
+        <CardContent className={classes.cardContent}>
+          <div>
+            {/* TODO Is name here the name of the project or language. */}
+            {/* Do I use name or display name */}
+            <Typography variant="h4">{name?.value}</Typography>
+            <Typography
+              variant="body2"
+              color="primary"
+              className={classes.location}
+            >
+              Tenron, Asia
+              {/* TODO I'm leaving this hard coded for the moment since I'm not sure */}
+              {/* if this is supposed to be a list of languages, one language, or no language */}
```

Comment:
This is a location. We just discovered yesterday that Languages need a list of them. So we are waiting for that to be implemented, but also the designs only show one which is tricky since there actually could be multiple.

Because of this, for now I would just leave it out completely.

Reply:
Leaving it out for now